.github/workflows/main.yml

```diff
@@ -2,11 +2,13 @@ name: h5z-zfp
 
 # Controls when the action will run. 
 #Triggers the workflow on push or pull requests.
+
+# TODO: REMOVE feat-cmake_tests before merging
 on: 
   push:
-    branches: [ master ]
+    branches: [ feat-cmake_tests, master ] 
```

Comment:
Just a reminder that you plan to remove this old feature branch name

Reply:
Changed.